### ISSUE TITLE
Coin catalog

### ISSUE THREAD

**Issue by shaimo** [graphprotocol/mission-control-curator#350]
I suggest everest, or a separate parallel site, would be used also as a global coin catalog, with each coin getting a unique global ID. Today various coin portals (coinmarketcap, coinpaprika, coingecko, etc) each use its own internal ID. This makes it messy e.g. to migrate from one portal to another (when using their API or whatever). 

Also exchanges provide coin information with their own proprietary IDs (e.g. when using their API) usually as just a symbol, which is ambiguous at best.

Everest could become the single source of truth for existing coins and their underlying information (symbol, contract in case of ERC20, usage, project they belong to, etc).

**Comment by yanivtal** [comment]
Yes! **Assets** should be tracked on a separate registry and linked to **Projects**.
